requirements.conda.yml

```diff
@@ -4,19 +4,20 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.6
-  - setuptools==45.1.0
-  - Click==7.0
-  - cython=0.29.15
-  - h5py=2.8.0
-  - matplotlib-base=3.1.3
-  - numpy=1.18.1
-  - opencv=4.2
-  - pillow=7.0.0
-  - pip=20.0.2
-  - pyyaml=5.3.1
-  - requests=2.23.0
+  - python=3.7
+  - setuptools<=45.1.0
+  - Click
+  - cython
+  - h5py
+  - matplotlib
+  - numpy
+  - opencv>=4.0
```

Comment:
why >= opencv ? What if a later version breaks something?

Reply:
We will specify minimum version number required for the package unless the code breaks, in which case we will create an issue and update the code. Until the code is fixed the version number for that particular package can be pinned.